userguide/tutorials/invoice_examples.adoc

```diff
@@ -0,0 +1,132 @@
+= Invoice Examples
+
+This document provides examples of different invoice scenarios. 
+
+== Prerequisites
+
+Ensure that you have gone through the https://docs.killbill.io/latest/userguide_subscription.html#components-invoicing[_Subscription User Guide_] and are familiar with the basic invoicing concepts.
+
+
+== Invoice Terminology - work in progress
+
+Whenever there is a change in the system (a recurring subscription billing, an external charge, a change in plan, and so on), the system generates an invoice. An invoice consists of one or more invoice items of different types. In addition, an invoice consists of several components like `balance`, `chargedAmount`, `paidAmount` and so on. Let us first understand some of these terms. 
```

Comment:
> In addition, an invoice consists of several components like `balance`, `chargedAmount`, `paidAmount` and so on. Let us first understand some of these terms. 

An invoice 'consists' of several invoice items but does not really consists of `balance`, `chargedAmount`, ... those are more like attributes, I think we should re-phrase this. Maybe our linguist expert @FlowWriter can help here?

Reply:
I've removed the invoice terminology section, these definitions can be added to the Glossary.